index/templates/index/index.html

```diff
@@ -12,8 +12,9 @@
             <div class="row">
                 <div class="col-md-6 my-auto text-center">
                     <h2 class="fs-1 fw-bolder">Welcome to United Events</h2>
+                    Your one-stop-shop for major Pride event-related information in the UK this summer<br>&#x1F308;&#x1F31E;&#x1F60E;
                     <p>
-                        Your go-to platform for major Pride event-related information in the UK this summer
+                        Connecting&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Participating&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Supporting&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Celebrating
```

Comment:
Why are there so many `&nbsp;` here? This is not acceptable in standard practice. If you want gaps, use either CSS or the built-in Bootstrap classes.

Reply:
Based on what I can see on the updated deployment, it seems you've understood WHY they are there - based on my knowledge & ability, "imperfectly complete">"perfectly incomplete".

There were NO winners here - I suffer from feeling throwing in the towel [angry, exasperated, exhausted & incredibly upset!], the rest of you suffer from me quitting [leaving you to pick up the broken pieces] and it ALL reflects VERY poorly on CI due to being exposed as villains for the education NOT being delivered.

EVERYBODY can BECOME winners IF CI learn from this and rectify it.

How is it that the first module in L3 "Programming Implementation" has absolutely NO mention whatsoever of ANY [let alone ALL!] of:

GITHUB
GITPOD
VS-CODE
BOOTSTRAP

????!

I'm only able to ask that now since, as a consequence of throwing myself in at the deep end, my eyes have been opened to see & know just how VERY much I don't [but desperately NEED to] know because it hasn't just not been taught; it hasn't even been mentioned despite all quite obviously being hugely fundamental to working out in the big wide bad wild world!

It's totally unacceptable that I had to drown in order to discover such sad & shameful reality!

Reply:
But remember that you have contributed a lot to the project in a lot of ways! I understand technology can be frustrating at times, but you learn from it gradually over time. I still have plenty of gaps in my knowledge too!